docs/getting_started/prepare_environment.md

```diff
@@ -32,6 +32,26 @@ sudo usermod -aG docker $USER
 
 **You will need to log out and lock back in** for user changes to take effect.
 
+You can also setup this environment on CBL-Mariner
+```bash
+# Install required dependencies.
+sudo dnf -y install git make tar wget curl rpm golang genisoimage python bison gawk glibc-devel binutils kernel-headers
```

Comment:
Andrew just updated our core docs as well: https://github.com/microsoft/CBL-Mariner/pull/5063

I'm thinking we may want to unify to a single source of truth (ie hyperlink back to the core repo for some of this).